p2p/net/mock/mock_test.go

```diff
@@ -14,8 +14,8 @@ import (
 	protocol "github.com/ipfs/go-libp2p/p2p/protocol"
 	testutil "github.com/ipfs/go-libp2p/testutil"
 
-	context "gx/QmacZi9WygGK7Me8mH53pypyscHzU386aUZXpr28GZgUct/context"
 	detectrace "github.com/jbenet/go-detect-race"
+	context "gx/QmacZi9WygGK7Me8mH53pypyscHzU386aUZXpr28GZgUct/context"
```

Comment:
why cant this be `ipfs/`? or `gx/ipfs/` ? (preserving paths is important. adding ipns support and other things will be possible. 

please please avoid breaking protocol naming. this is the kinda crap that makes it hard to nest + reuse protocols.